rwengine/src/audio/MADStream.hpp

```diff
@@ -10,7 +12,13 @@
 #include <stdio.h>
 #include <sys/mman.h>
 #include <sys/stat.h>
+#if defined(HAVE_UNISTD_H)
 #include <unistd.h>
+#elif defined(HAVE_IO_H)
+#include <io.h>
+#else
+#error "Need an alternative for unistd.h and io.h"
+#endif
```

Comment:
imo these 2 should be implied for unix like platforms and seperate checks / implementations should be added for windows imo